### PR TITLE
[TRANSLATION] EN to KO: render-blocking-css

### PR DESCRIPTION
**Pull request by SungwonKim**
I signed the CLA.

### CODE REVIEW

**Comment by cwdoh** [comment]
:+1:

**Comment by SungwonKim** [comment]
I signed it.

**Comment by PaulKinlan** [comment]
Thank you!

File: src/content/ko/fundamentals/performance/critical-rendering-path/render-blocking-css.markdown [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+title: "렌더링을 블록 하는 CSS"
+description: "기본적으로 CSS는 렌더링을 블록 시키는 자원으로 여겨집니다, 어떻게 하면 렌더링이 블록되는 것을 막을 수 있는 지 배우세요."
```
Comment (cwdoh) [inline src/content/ko/fundamentals/performance/critical-rendering-path/render-blocking-css.markdown]: suggestion: 배우세요 -> 알아봅시다.

**Pull request merged**